src/runtime/auto-prompt-manager.js

```diff
@@ -190,17 +192,23 @@ export class AutoPromptManager {
     params
   ) {
     // Override autoPromptType if it is undefined.
-    params.autoPromptType =
-      params.autoPromptType ||
-      this.getAutoPromptType_(article?.audienceActions?.actions);
+    params.autoPromptType ??= this.getAutoPromptType_(
+      article?.audienceActions?.actions
+    );
+
+    // Override isClosable if isAccessibleForFree is defined.
```

Comment:
Is this code checking whether `isAccessibleForFree` is defined? It seems like values of `false` and `undefined` would be treated the same

Reply:
In this case I'm casting it to a bool with `!!`, so it should convert undefined to false. That should be fine here, but maybe we would be better off just checking `isLocked`... I'll look at it again.

Reply:
Looked at it again, pretty sure I'm wrong. Properly checking if it's undefined first.

Reply:
Sweet, I'm taking another look